plugins/modules/gitlab_project_members.py

```diff
@@ -89,6 +89,26 @@
     elements: str
     choices: ['guest', 'reporter', 'developer', 'maintainer']
     version_added: 3.7.0
+  gitlab_group:
+    description:
+      - A list of id or the full path of the group to share/unshare the project with.
```

Comment:
```suggestion
      - A list of IDs or the full paths of the groups to share/unshare the project with.
```

Reply:
I got it from https://docs.ansible.com/ansible/latest/collections/community/general/gitlab_project_module.html#parameter-group, I can change it to ID